src/translations/main_it.ts

```diff
@@ -624,22 +624,23 @@ Assicurati di aver selezionato la directory corretta dei profili.</translation>
     </message>
     <message>
         <source>Fatal error: %1
-
+            
```

Comment:
```suggestion

```
I just noticed you edited the `source` here. I think you meant to edit your translation.